packages/cozy-mespapiers-lib/src/components/Search/search.js

```diff
@@ -1,40 +1,34 @@
 import { Document } from 'flexsearch'
 
-import { models } from 'cozy-client'
+import { themesList } from 'cozy-client/dist/models/document/documentTypeData'
+import { isFile, hasQualifications } from 'cozy-client/dist/models/file'
 
-import { CONTACTS_DOCTYPE } from '../../doctypes'
-
-const {
-  document: {
-    themes: { themesList }
-  },
-  file: { isFile }
-} = models
+import { FILES_DOCTYPE, CONTACTS_DOCTYPE } from '../../doctypes'
 
 export const index = new Document({
```

Comment:
I wonder if it's not worth adding some comments explaining the basic concepts here, like: 
```
The index document will store (_id, _type) couples for each document having the declared indexed fields, coming from both `io.cozy.files` and `io.cozy.contacts`. 
Note some special fields are prefixed with `flexsearchProps` as they are not actual doc attributes, but they are dynamically built when a doc is added to the index.
`tag` is a special concept allowing to search by category, i.e. "identity", "family", etc.
```

Reply:
fait dans le dernier commit, merci pour le texte 👍